### PR TITLE
Spaces in shellout evaluate properly

### LINKED ISSUE #492: spaces in backtick shellout don't work
```sh
$ cat backtick-test 
bind-key `echo s` sort
$ newsboat -C backtick-test
Starting newsboat 2.13.0...
Loading configuration...Error while processing command `bind-key `echo s` sort' (backtick-test line 1): `sort' is not a valid context
```

Expected behavior:
```bind-key `echo s` sort``` expands to ```bind-key s sort``` and is valid

Initial investigation shows that `tokenize_quoted` is splitting arguments out by spaces, which are then getting individually parsed by `evaluate_backticks`.

Therefore, only one-word shellouts work

### PR DESCRIPTION
**Pull request by mjsir911**
Fixed #492

### CODE REVIEW

**Comment by mjsir911** [comment]
One foreseeable problem: I've moved `evaluate_backticks` outside of the `try {}` block, so errord commands might behave differently now

File: test/configparser.cpp [hunk 1]
```diff
@@ -44,6 +50,14 @@ TEST_CASE("evaluate_backticks replaces command in backticks with its output",
 				"a single literal backtick: \\`") ==
 			"a single literal backtick: `");
 	}
+	SECTION("commands with space are evaluated by backticks")
+	{
+		ConfigParser cfgparser;
+		KeyMap keys(KM_NEWSBOAT);
+		cfgparser.register_handler("bind-key", &keys);
+		REQUIRE_NOTHROW(cfgparser.parse("data/config-space-backticks"));
+		REQUIRE(keys.get_operation("s", "all"));
```
Comment (Minoru) [inline test/configparser.cpp]: Implicit casting confused me for a moment here: this reads like `get_operation()` returns a `bool`, and this doesn't align with the method name. I suggest being explicit: `REQUIRE_FALSE(keys.get_operation("s", "all") == OP_NIL)`.

**Comment by Minoru** [comment]
I checked and neither `evaluate_backticks` nor any of `utils` functions it call throw anything, so we should be good.

If you edit that test a bit, this should be good for merge. Thanks for the work!

**Comment by Minoru** [comment]
Thank you! I'll merge once CI passes.

**Comment by Minoru** [comment]
Thanks, both for the report and the bugfix!

**Pull request merged**